src/modules/collections/routes/browse/browse.vue

```diff
@@ -112,6 +107,26 @@
 			:filters.sync="filters"
 			:search-query.sync="searchQuery"
 		/>
+
+		<template #drawer>
+			<drawer-detail icon="info_outline" :title="$t('information')" close>
+				<v-detail>
+					<template #title>{{ $t('Need Help?') }}</template>
```

Comment:
Wrong translation here

```suggestion
					<template #title>{{ $t('need_help') }}</template>
```

also make sure to add this to en-us/index.json